### ISSUE TITLE
The remote server returned an error: (301) Moved Permanently.

### ISSUE THREAD

**Issue by manuc66** [ghorsey/OpenGraph-Net#25]
**Describe the bug**
OpenGraph fetching thows where the target url return a 301 status code

**To Reproduce**
```cs
await OpenGraph.ParseUrlAsync("https://news.google.com/__i/rss/rd/articles/CBMigwFodHRwczovL3d3dy5zdWRpbmZvLmJlL2lkMzEzNzM4L2FydGljbGUvMjAyMS0wMS0yMi9sZS1jb21pdGUtZGUtY29uY2VydGF0aW9uLWRlYnV0ZS1kZS1ub3V2ZWxsZXMtbWVzdXJlcy1wbHVzLXJlc3RyaWN0aXZlcy1xdWlkLWRlc9IBAA?oc=5")
```

If this url does not work in the future, this kind of url come from : https://news.google.com/news/rss/?ned=fr_be&gl=BE&hl=fr

**Expected behavior**
I would have expected it fetch the OpenGraph located at : 

https://www.sudinfo.be/id313738/article/2021-01-22/comite-de-concertation-de-nouvelles-mesures-pour-lutter-contre-le-covid-une

**Screenshots**
![image](https://user-images.githubusercontent.com/5827164/105559189-d20f2580-5d10-11eb-9e2b-aec7e548839c.png)

**Desktop (please complete the following information):**
 - OS: Linux
 - .Net 5.0 
 - Version : `<PackageReference Include="OpenGraph-Net" Version="3.2.4" />`

**Comment by manuc66** [comment]
It seems that the fix #17 do not works for this kind of scenario

**Comment by ghorsey** [comment]
@manuc66  this will go out with version 3.2.5 today.

**Comment by ghorsey** [comment]
I just published v3.2.6 (I updated some package info that I missed) that has this fix.

Docs updated, see: https://raw.githubusercontent.com/ghorsey/OpenGraph-Net/main/NUGET.md

cc: @manuc66

**Comment by manuc66** [comment]
Thanks !
